fe/src/components/Pagination.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const Pagination = (props) => {
+  const pageNumber = [];
+
+  for (let i = 1; i <= Math.ceil(props.totalItems / props.itemsPerPage); i++) {
+    pageNumber.push(i);
+  }
```

Comment:
- ahh you don't need a loop for this one. just get the `totalPageItems`
- and then loop the 
```suggestion
  const totalPageItems = Math.ceil(props.totalItems / props.itemsPerPage;
  
  return for(let item=0; item <= totalPageItems; item) {
    <li>...
  }
```

Reply:
Hi sir  @escbooster12-dev I tried it again, this is what happened

![image](https://user-images.githubusercontent.com/115448429/202989518-f0b057c8-e6f5-4510-bd52-8cdd329065f8.png)
![image](https://user-images.githubusercontent.com/115448429/202989479-87a7e8e1-4aad-47d2-88d9-85bcbfb3c958.png)